modules/ROOT/pages/installation.adoc

```diff
@@ -7,3 +7,472 @@ endif::[]
 Install API Community Manager with these instructions.
 
 Installed as a Salesforce package.
+
+API Community Manager 
+Setup Guide
+ (RC Version)
+
+Organization Installation Requirements - prereqs???
+Installing API Community Manager 
+Update API Community Manager - move this???
+Create a Community
+API Community Manager Components
+Publish API Community Manager
```

Comment:
remove this from here - instead, add a section "Customizing the look&feel and publishing your community" that just points the reader to the `brand-intro.adoc` file (link is in nav but didn't create the file yet).